biztalk/core/how-to-configure-transport-advanced-options-for-a-send-port.md

```diff
@@ -67,11 +74,12 @@ Use the BizTalk Server Administration console to configure transport advanced op
     |**Priority**|Set the priority of the resend attempt.|  
     |**Ordered delivery**|Select this check box to send messages in order of receipt.|  
     |**Stop sending subsequent messages on current message failure**|Select this check box to stop sending subsequent messages that follow a failed message. This option is available only when the **Ordered delivery** option is selected.|  
+    |**Enforce order across outbound locations**|This property is only visible for dynamic send ports. Select this check box to enforce ordered delivery across all outbound locations for a given transport type. This option is available only when the **Ordered delivery** option is selected.|  
```

Comment:
Do you want to move "This property is only visible for dynamic send ports" after the "Select this check box..."?
like:
This property is only visible for dynamic send ports and only available when ordered delivery is checked.

Reply:
I want to call out first that this property is only for dynamic ports. User does not have to read the rest of the description if they are not using a dynamic port.